OMPython/__init__.py

```diff
@@ -238,8 +239,11 @@ def sendExpression(self, command):
             self._omc = None
             return result
           else:
-            answer = OMTypedParser.parseString(result)
-            return answer
+            try:
+              answer = OMTypedParser.parseString(result)
+              return answer
+            except:
```

Comment:
Ah, so you are saying that if parseString throws an exception, you assume it's because `result` contains only a compiler error message, and not because of otherwise malformed compiler output, so you return the compiler error message instead of the parsed `answer`, right?

Unfortunately, I'm on Python 3, so not using OMPython (yet).